biogeochem/EDPhysiologyMod.F90

```diff
@@ -1959,7 +1960,7 @@ subroutine assign_cohort_SP_properties(currentCohort,htop,tlai,tsai,parea,init,l
          currentCohort%n, currentCohort%canopy_layer,               &
          canopylai,currentCohort%vcmax25top )
 
-    if( abs(currentCohort%treelai-check_treelai).gt.1.0e-12)then !this is not as precise as nearzero
+    if (abs(currentCohort%treelai - check_treelai) > area_error) then !this is not as precise as nearzero
```

Comment:
This also increases the tolerance of this check.  It might be good to check to see if this changes answers during testing.

Reply:
Sure can do